bower.json

```diff
@@ -15,7 +15,7 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "ignore": [],
   "dependencies": {
-    "polymer": "polymer/polymer#^1.0.0",
+    "polymer": "polymer/polymer#master",
```

Comment:
Should not need to use #master now.  [Polymer v1.6.0](https://github.com/Polymer/polymer/releases/tag/v1.6.0) is tagged.